### PR TITLE
common: tests: move tests to top level directory

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by arnout — COMMENTED** [review]
I'm not sure why splitting the moves off in a separate commit is good. It's more difficult to review and it makes it un-bisectable, and it's also more confusing when looking at the history.

Also, I don't think it's appropriate for the tests-runner Dockerfile to move to the top level - it should stay in support/docker - though maybe better in support/docker/tests-runner.

File: .gitlab-ci.yml
Comment (rmelotte) [inline .gitlab-ci.yml]: `pushd` doesn't work because this jobs uses busybox's sh. I don't think it's needed though.
If possible I think `image-pull.sh` should be called from the root dir directly. I don't think we need to keep `image-build.sh`.

**Review by rmelotte — COMMENTED** [review]
Only one small comment for now.
Also since we removed Travis entirely, we could remove `.travis.yml` so that we don't have to update it everytime.

File: .gitlab-ci.yml
Comment (rmelotte) [inline .gitlab-ci.yml]: As we now provide this variable from the Gitlab schedules ([PR](https://github.com/prplfoundation/prplMesh/pull/730) not yet merged), we will need to update the value there.
When you merge this, please change it on Gitlab or notify me otherwise the nightly tests will fail.

File: .gitlab-ci.yml
Comment (rmelotte) [inline .gitlab-ci.yml]: As I mentioned in my previous comment, please use `tools/docker/image-pull.sh` directly to avoid changing dirs around.

File: .travis.yml
Comment (rmelotte) [inline .travis.yml]: `.travis.yml` can be removed.

**Review by rmelotte — APPROVED** [review]
Minor comments but please take them into account.

I also think the 2 commits should be squashed, because if you take them separately they break the tests.
Approving so that you don't have to wait for a re-review after you made the changes.

File: CONTRIBUTING.md
Comment (arnout) [inline CONTRIBUTING.md]: This reference to .travis.yml should have been removed as well.

**Review by arnout — APPROVED** [review]
The removal of .travis.yml should have been a separate commit.

I'll do that and merge.

**Pull request merged**
